examples/src/main/scala/caliban/finch/ExampleApp.scala

```diff
@@ -13,13 +13,12 @@ import zio.clock.Clock
 import zio.console.Console
 import zio.interop.catz._
 import zio.stream.ZStream
-import zio.{ DefaultRuntime, Task, URIO }
-
+import zio.{ Runtime, Task, URIO }
 import scala.language.postfixOps
 
 object ExampleApp extends App with GenericSchema[Console with Clock] with Endpoint.Module[Task] {
 
-  implicit val defaultRuntime = new DefaultRuntime {}
+  implicit val runtime = Runtime.unsafeFromLayer(Console.live ++ Clock.live)
```

Comment:
@adamgfraser can confirm it's the right way to do if I need to pass a `Runtime[Console with Clock]` and don't control the calls to `unsafeRun`?

Reply:
@ghostdogpr Can you do `val runtime = Runtime.default` and the use `provideLayer` to provide the effect with its required environment before running it?

Reply:
No because the `unsafeRun` is not done here, it will be done by the framework (here Finch but same with Akka Http) for each request.